### DISCUSSION TITLE
Can't modify the Buffer Size

### DISCUSSION THREAD

**Discussion by Ardalien** [apohl79/audiogridder#270] [Q&A] [answered]
In the alpha versions I could modify the buffer size according to what I needed. from the smallest to the largest and in most of the times it worked ok. Now in the stable 1.0 version nothing doesn't happen when I try to select different buffer sizes from the plugin, It stays to 8 blocks. From all the Beta version I could only manage to run  1.7 server on  OSX ( Sierra ) , 1.9 and 10 have the plugin list empty on the server  after scanning and none appear in the client plugin. Am I doing something wrong ? Can I make the 1.9 and 10 server work correctly on OSX Sierra? . Is the buffer sized locked or should it be switchable ?

**Comment by Ardalien** [comment]
I am running the Client plugin in Ableton Live 10 on OSX Sierra .

**Comment by CaBuffalo** [comment]
Hi, Ardalien:

I'm not running Sierra. I have Catalina. Yes, the buffer size are all switchable. Wondering if you try the latest Beta 10? You can select 1 block if it works.

**Comment by Ardalien** [comment]
I will try with beta 10 between Client Osx and Server on windows. But I need the server running on my other Macbook and the beta 10 server doesn't remember the plugins ( The plugin list is empty ) after scan on the OSX. It also doesn't save any setting I do in the server settings panel. Every time I open it, it starts and scans all the plugins and every setting in the penal is default. How can I fix this ?

**Comment by CaBuffalo** [comment]
My set up is Logic + Windows 10 server. I can see all of the plugins I
installed.

On your OSX server, did you install all VST2/3 and AU format ? It's very
strange that your list is empty. Also, what I can think of is if you give
AG permission to access your security/privacy settings.

On Fri, Dec 18, 2020 at 9:25 AM Ardalien <notifications@github.com> wrote:

> I will try with beta 10 between Client Osx and Server on windows. But I
> need the server running on my other Macbook and the beta 10 server doesn't
> remember the plugins ( The plugin list is empty ) after scan on the OSX. It
> also doesn't save any setting I do in the server settings panel. Every time
> I open it, it starts and scans all the plugins and every setting in the
> penal is default. How can I fix this ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/issues/270#issuecomment-748217431>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FJIJQND7V6RHZQPSLSVOGBBANCNFSM4VBFNKJQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment, marked as answer]
Try wiping all server configs. From 9 to 10 the locations changed. See here:

https://audiogridder.com/server/#config

Wipe pre beta10 and beta10 files. Than let the server rescan. This is like a fresh install.

**Comment by Ardalien** [comment]
Apohl79   I can't find the locations on the OSX to wipe the files.

**Comment by apohl79** [comment]
Another note regarding config files: The new beta10 config structure conflicts with the old plugin config file name, as I moved all AG related files into a directory. The name of that directory equals the name of the plugin config previously. 

Means, if you downgrade the plugin, you _have_ to remove the config directly, or the pre-beta10 plugin  can’t store any settings.

**Comment by apohl79** [comment]
Try cmd+shift+. in finder to make hidden files visible.

**Comment by Ardalien** [comment]
I see all the hidden files but I don't know where the location of the files are.

**Comment by Ardalien** [comment]
I copied  ~/.audiogridder/audiogridderserver.cfg    in the  command+shift+g  "go to the folder" finders command. and also this  ~/.audiogridderserver .  Doesn't take me anywhere.

**Comment by Ardalien** [comment]
I have found a file only here ~/.audiogridderserver . Sorry, I will delete it now and try the beta 10

**Comment by apohl79** [comment]
Ok sorry, i don’t follow. :-)

Open the terminal app and run:

rm -rf ~/.audiogridder*

This wipes all AG files of all versions.

**Comment by CaBuffalo** [comment]
It's under your username. for example, mojave>users>mike

On Fri, Dec 18, 2020 at 9:56 AM Ardalien <notifications@github.com> wrote:

> I see all the hidden files but I don't know where the location of the
> files are.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/issues/270#issuecomment-748231377>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6B6MJHMO3TKUMBMEN3SVOJUHANCNFSM4VBFNKJQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Ardalien** [comment]
I have found them and deleted the files. I will try the beta 10 now. Thank you

**Comment by Ardalien** [comment]
Ok, thank you guys. The buffers size switch works now and I can run beta 10 both on win and osx.

**Comment by CaBuffalo** [comment]
Awesome, cheers. 👍
